__tests__/boot.ts

```diff
@@ -129,6 +129,7 @@ const LOGGED_IN_BODY = {
     mastodon: null,
     language: undefined,
     isPlus: false,
+    plusProvider: null,
```

Comment:
Can you add a test that it is also returned for logged user boot?

Reply:
👍